packages/eslint-config/package.json

```diff
@@ -1,8 +1,9 @@
 {
 	"name": "@rocket.chat/eslint-config",
-	"version": "0.5.0",
-	"description": "Rocket.Chat's JS ESLint config",
-	"devDependencies": {
+	"version": "0.5.2",
+	"description": "Rocket.Chat's JS/TS ESLint config",
+	"dependencies": {
+		"@babel/core": "^7.20.7",
 		"@babel/eslint-parser": "^7.19.1",
 		"@types/prettier": "^2.6.3",
```

Comment:
`@types/*` dependencies must be dev-dependencies, but in this case (`@types/prettier`) it not even be needed here.